project_task_code/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     'name': 'Sequential Code for Tasks',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.0.1',
```

Comment:
@alexey-pelykh No need to Update version, it's Code Error And you DIrectly Fixed that.

It's Improvement Not added Any functionality in Exit code.

Reply:
It's a bug fix for published module, version bump is required